tests/Unit/controllers/channel.test.js

```diff
@@ -0,0 +1,257 @@
+const errorsFactory = require("../../../src/utils/error/errors");
```

Comment:
Can you add some unit test for catching general errors. Just make sure that any errors get handled properly

Reply:
not too sure what u mean by that

Reply:
I would like to see a test where you cause a random error and see that the code catches it properly. The purpose of this is to ensure there is always error catching.

Reply:
From discussion, we determined this test is not necessary